WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -678,3 +678,36 @@ d_vasconic_pagan_reformed = {
 	
 	dynasty_title_names = no
 }
+
+d_priscillianism = {
+	color={ 153 153 153 }
+	color2={ 220 220 0 }
+	
+	capital = 198 # Valladolid
+	
+	title = "POPE"
+	foa = "POPE_FOA"
+	
+	short_name = yes
+	
+	allow = {
+		always = no # Only by special decision
+	}
+
+	gain_effect = {
+		add_trait = vegetarian
+	}
+
+	# Always exists
+	landless = yes
+	
+	# Controls a religion
+	controls_religion = priscillianism
+	
+	religion = priscillianism
+	
+	# Cannot be held as a secondary title
+	primary = yes
+	
+	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
+}
```

Comment:
You didn't move it back, you moved it to another file. In the master it is in landed_titles on line 40876.